### PR TITLE
WIP: Cart management

### PR DESCRIPTION
**Pull request by way2nizam**
screenshot (1)

### CODE REVIEW

File: styles/cart.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+.price__total {
+  font-size: 1.5rem;
+  font-weight: 700;
+}
+.price__benefit {
+  font-weight: 500;
+}
+
+.price-btn {
```
Comment (spraveenofficial) [inline styles/cart.css]: Remove this empty style.

**Review by spraveenofficial — APPROVED** [review]

**Review by ishapatell1 — APPROVED** [review]
Looks Good to merge..

**Review by chanchal-panpaliya — APPROVED** [review]
code looks good .

File: styles/cart.css (hunk 1 above)
Comment (way2nizam) [reply styles/cart.css]: done, thanks

**Pull request merged**